HISTORY.rst

```diff
@@ -9,7 +9,12 @@ History
 18.5.6.dev0
 ---------------------
 
-    
+* Add missing markupsafe dependency (thanks to @mbargull).
```

Comment:
Thanks for the acknowledgement, but no need to do it twice -- already done in `18.5.5` :wink:

Reply:
Good catch - thanks.